compiler/src/dmd/dmodule.d

```diff
@@ -361,6 +361,7 @@ extern (C++) final class Module : Package
     int needmoduleinfo;
     int selfimports;            // 0: don't know, 1: does not, 2: does
     Dsymbol[void*] tagSymTab;   /// ImportC: tag symbols that conflict with other symbols used as the index
+    Symbol* isym;               /// Import version of csym used for DllImport on Windows
```

Comment:
Need to stop accepting dmd-specific back end symbols in the front-end. There are plenty of ways to keep compiler-specific information contained to the code generator.

Reply:
This is the existing solution for toImport for a different node and it is aligned with msym that is defined in a parent.

If there is a way to solve this assuming I can get this to go green, please do tell me what to do.